Tests/SentryTests/SentrySDKTests.swift

```diff
@@ -219,6 +219,38 @@ class SentrySDKTests: XCTestCase {
         }
     }
     
+    func testPerformanceOfConfigureScope() {
+        func buildCrumb(_ i: Int) -> Breadcrumb {
+            let crumb = Breadcrumb()
+            crumb.message = String(repeating: String(i), count: 100)
+            crumb.data = ["some": String(repeating: String(i), count: 1_000)]
+            crumb.category = String(i)
+            return crumb
+        }
+        
+        SentrySDK.start(options: ["dsn": TestConstants.dsnAsString])
+        
+        SentrySDK.configureScope { scope in
+            let user = User()
+            user.email = "someone@gmail.com"
+            scope.setUser(user)
+        }
+        
+        for i in Array(0...100) {
+            SentrySDK.configureScope { scope in
+                scope.add(buildCrumb(i))
+            }
+        }
+        
+        self.measure {
```

Comment:
what does the measuring method do?

Reply:
It measures the performance of the code, see https://developer.apple.com/documentation/xctest/xctestcase/1496290-measure. You can then set a baseline with Xcode and if the code inside `measure` gets slower the test fails.

Reply:
I didn't understand how this work. What's the default time of which it measures against?
If it takes 1 second, does it fail?

Did this test fail (consistently) before the change was introduced?